api/src/chat/services/bot.service.ts

```diff
@@ -252,13 +252,13 @@ export class BotService {
           assign_labels: [],
           trigger_labels: [],
           attachedBlock: undefined,
-          category: undefined,
+          category: undefined as any,
           previousBlocks: [],
         };
         convo.context.attempt++;
         fallback = true;
       } else {
-        convo.context.attempt = 0;
+        if (convo.context) convo.context.attempt = 0;
```

Comment:
`convo.context` should be undefined or null. I believe there's alway a default value in the schema, right ?

Reply:
Yes the context will always get a default value
![image](https://github.com/user-attachments/assets/15491c57-7276-4300-9926-9b2462a19b70)